Using Event as an entity name results in a compile error

##### **Overview of the issue**

Create an app with the following JDL and it'll fail to compile the frontend.

```
application {
 config {
   baseName space,
   applicationType monolith,
   authenticationType oauth2,
   packageName org.jhipster.space,
   prodDatabaseType postgresql,
   testFrameworks [protractor]
 }
 entities *
}

entity Event {
name String required
 date LocalDate required
 description TextBlob required
 photo ImageBlob required,
 type EventType required
}

entity Mission {
name String required
 description String
}

enum EventType {
 Launch,
 Landing
}

relationship OneToOne {
 Event{mission(name)} to Mission
}

paginate Event with infinite-scroll
paginate Mission with pagination
```

Error:

```
[INFO] Webpack: Finished after 10.422 seconds.
[INFO]
[INFO]  ERROR  Failed to compile with 1 errors8:36:27 AM
[INFO]
[INFO]  error
[INFO]
[INFO] src/main/webapp/app/entities/event/event-update.component.ts:96:35 - error TS2345: Argument of type 'import("/Users/mraible/app/src/main/webapp/app/shared/model/event.model").Event' is not assignable to parameter of type 'Event'.
[INFO]   Type 'Event' is missing the following properties from type 'Event': bubbles, cancelBubble, cancelable, composed, and 17 more.
[INFO]
[INFO] 96     this.dataUtils.loadFileToForm(event, this.editForm, field, isImage).subscribe(null, (err: JhiFileLoadError) => {
[INFO]                                      ~~~~~
[INFO]
[INFO]
[INFO]    5 modules
[INFO]
[INFO] ERROR in src/main/webapp/app/entities/event/event-update.component.ts:96:35 - error TS2345: Argument of type 'import("/Users/mraible/app/src/main/webapp/app/shared/model/event.model").Event' is not assignable to parameter of type 'Event'.
[INFO]   Type 'Event' is missing the following properties from type 'Event': bubbles, cancelBubble, cancelable, composed, and 17 more.
[INFO]
[INFO] 96     this.dataUtils.loadFileToForm(event, this.editForm, field, isImage).subscribe(null, (err: JhiFileLoadError) => {
[INFO]
```

##### **Motivation for or Use Case**

I tried to create an app with `Event` as an entity and it fails to work. I'd expect a warning that this is a reserved word rather than it failing after I try to build my app.

##### **Suggest a Fix**

Add `Event` to the list of keywords that we look for and warn about.

INFO! Using JHipster version installed locally in current project's node_modules
INFO! Executing jhipster:info
##### **JHipster Version(s)**

```
space@0.0.1-SNAPSHOT /Users/mraible/app
└── generator-jhipster@6.10.1 

```


##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**


<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "authenticationType": "oauth2",
    "cacheProvider": "ehcache",
    "clientFramework": "angularX",
    "serverPort": "8080",
    "serviceDiscoveryType": false,
    "skipUserManagement": true,
    "baseName": "space",
    "buildTool": "maven",
    "databaseType": "sql",
    "devDatabaseType": "h2Disk",
    "enableHibernateCache": true,
    "enableSwaggerCodegen": false,
    "enableTranslation": true,
    "jhiPrefix": "jhi",
    "languages": ["en", "fr"],
    "messageBroker": false,
    "nativeLanguage": "en",
    "prodDatabaseType": "postgresql",
    "searchEngine": false,
    "skipClient": false,
    "testFrameworks": ["protractor"],
    "websocket": false,
    "applicationType": "monolith",
    "packageName": "org.jhipster.space",
    "packageFolder": "org/jhipster/space",
    "jhipsterVersion": "6.10.1",
    "creationTimestamp": 1596637920237,
    "skipServer": false,
    "clientPackageManager": "npm",
    "clientTheme": "none",
    "clientThemeVariant": "",
    "useSass": true,
    "embeddableLaunchScript": false,
    "entitySuffix": "",
    "dtoSuffix": "DTO",
    "otherModules": [],
    "blueprints": []
  },
  "entities": ["Event", "Mission"]
}

</pre>
</details>


##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<details>
<summary>JDL entity definitions</summary>

<pre>
entity Event {
  name String required,
  date LocalDate required,
  description TextBlob required,
  photo ImageBlob required,
  type EventType required
}
entity Mission {
  name String required,
  description String
}
enum EventType {
  Launch,
  Landing
}

relationship OneToOne {
  Event{mission(name)} to Mission
}

paginate Event with infinite-scroll
paginate Mission with pagination

</pre>
</details>


##### **Environment and Tools**

openjdk version "11.0.5" 2019-10-15
OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.5+10)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 11.0.5+10, mixed mode)

git version 2.24.3 (Apple Git-128)

node: v12.16.3

npm: 6.14.5

yeoman: 3.1.1

yarn: 1.22.4

Docker version 19.03.8, build afacb8b

docker-compose version 1.25.5, build 8a1c60f6

INFO! Congratulations, JHipster execution is complete!

##### **Browsers and Operating System**

<!-- What OS are you on? is this a problem with all browsers or only IE8? -->

-   [x] Checking this box is mandatory (this is just to show you read everything)

<!-- Love JHipster? Please consider supporting our collective:
👉  https://opencollective.com/generator-jhipster/donate -->

--- a/generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management-update.component.ts.ejs
+++ b/generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management-update.component.ts.ejs
@@ -237,7 +237,7 @@ _%>
         this.dataUtils.openFile(contentType, base64String);
     }
 
-    setFileData(event: Event, field: string, isImage: boolean): void {
+    setFileData(event: any, field: string, isImage: boolean): void {
         this.dataUtils.loadFileToForm(event, this.editForm, field, isImage).subscribe(null, (err: JhiFileLoadError) => {
             this.eventManager.broadcast(
                 new JhiEventWithContent<AlertError>('<%= angularAppName %>.error', { <% if (enableTranslation) { %>...err, key: 'error.file.' + err.key<% } else { %>message: err.message<% } %> })

